recipes/fs.sshfs/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set name = "fs.sshfs" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e9f218a382c8a82c9f639d1c0e3a622ed005fed03923966621bafbfe0da12f4
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - skip: true  # [win]
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - fs
+    - six
+    - paramiko
+    - configparser
+
+test:
+  imports:
+    - fs.sshfs
+
+about:
+  home: https://github.com/althonos/fs.sshfs
+  license: LGPL v2.1
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
+  summary: 'Pyfilesystem2 over SSH using paramiko'
+  description: >
+    Usage
```

Comment:
I strongly encourage you do make this shorter. No need to replicate docs in the description.